### PR TITLE
Add simple collision and out of bounds detection.

### LINKED ISSUE #12: Collision detection
When #11 has been implemented, a simple collision detection should be implemented so the foxes disappear when they run in to each other.

### PR DESCRIPTION
**Pull request by dlebech**
This commit adds simple collision and out-of-bounds detection to the
server game loop.

Bonus:
- Use a class for the Zombie on the server :-)

Closes #12

### CODE REVIEW

**Comment by andreasschmidtjensen** [comment]
👍

**Comment by andreasschmidtjensen** [comment]
👯 :trollface:

File: lib/controller.js [hunk 1]
```diff
@@ -108,9 +108,35 @@ Game.prototype.update = function(elapsed) {
     player.update(elapsed);
   }
 
-  for (var key in this.zombies) {
-    var zombie = this.zombies[key];
+  const toDelete = []
```
Comment (dlebech) [inline lib/controller.js]: Whoops

**Comment by thomicdk** [comment]
:+1:

**Pull request merged**